src/common/pony/detail/atomics.h

```diff
@@ -8,10 +8,12 @@
 #  error "Unsupported platform"
 #endif
 
-#ifndef __cplusplus
+ #ifndef __cplusplus
 #  include <stdalign.h>
 #endif
 
+#include <platform.h>
```

Comment:
The problem here is that `platform.h` is a private header while `atomics.h` is a public one. Depending on how much of `platform.h` is needed, I'd suggest to either copy/paste the relevant parts into `atomics.h` or to make a new `platform_public.h` file which would be included by both `platform.h` and `atomics.h`, and installed alongside of `pony.h` and `atomics.h`.